addons/sourcemod/scripting/vip/UTIL.sp

```diff
@@ -299,7 +299,7 @@ void UTIL_ADD_VIP_PLAYER(int iAdmin = 0,
 	{
 		GetClientName(iTarget, SZF(szQuery));
 		g_hDatabase.Escape(szQuery, SZF(szName));
-		iAccountID = GetSteamAccountID(iTarget);
+		iAccountID = GetSteamAccountID(iTarget, false);
```

Comment:
also check the return value if its 0, throw an error:
- https://sourcemod.dev/#/clients/function.GetSteamAccountID